rubrics/services/src/main/webapp/sass/sakai-rubrics.scss

```diff
@@ -0,0 +1,314 @@
+// #sakai-rubrics-manager
```

Comment:
Hi, Earl. We will have to discuss this as these styles are part of the [Web Components](https://www.webcomponents.org/), which are [intended to be encapsulated (HTML, CSS and JavaScript as a reusable unit)](https://developers.google.com/web/fundamentals/getting-started/primers/shadowdom). With [Shadow DOM](https://www.w3.org/TR/shadow-dom/) or [Shady DOM](https://www.polymer-project.org/1.0/blog/shadydom) the [components inherit some core theme styles from the container](https://www.webcomponents.org/specs#styling) but are meant provide component specific styles, and to not leak component specific styles to anything else on the page.

There may be parts of the styling that make sense to move into Morpheus, but in general, use of Shadow/Shady DOM with web components is a new paradigm that should slightly change the way styling is handled in Sakai. Central theme aspects will stay in Morpheus, but there should be a place for component specific styles to stay directly with the components, especially given that these components can and should be able to be used and rendered completely outside of Sakai. The Rubrics services as designed can actually integrate and work with any LMS platform that wants to add rubrics functionality. There are a few clean-up items, particularly in the view layer to make this a reality which were deferred given that this is targeted for Sakai, but it is not far off. Diego is scheduling a meeting and I am available tomorrow so we can discuss more at that point.